packages/worker/src/worker-options.ts

```diff
@@ -444,10 +445,9 @@ export interface WorkerOptions {
    * guarantees, please consider using local activities instead. For use cases that require
    * _exactly-once_ or _at-most-once_ execution guarantees, please consider using regular activities.
    *
-   * The SDK itself may register sinks functions required to support workflow features. At the moment, the only such
-   * sink is 'defaultWorkerLogger', which is used by the workflow context logger (ie. `workflow.log.info()` and
-   * friends); other sinks may be added in the future. You may override these default sinks by explicitely registering
-   * sinks with the same name.
+   * The SDK itself may register sinks functions required to support workflow features. The name,
+   * signature and semantic of these sinks functions is considered an internal detail and may change
+   * in the future without notice. Please do not use them directly nor override them.
    */
   sinks?: InjectedSinks<any>;
```

Comment:
Should we perhaps put the internal sinks in a different, unexposed field so we don't need this disclaimer in the docstring?

Reply:
No. The reason behind that disclaimer is that previously and until now, user overriding our default sinks (specifically `defaultWorkerLogger`) was allowed, documented, and even encouraged. So this is really a change of direction. Therefore, I want to preserve backward compatibility for code that was using that approach, but strongly discourage that practice going forward.

New internal sinks that we'll add from now on will use names that clearly indicate they are meant for internal usage only (eg. `__temporal_metrics` or similar), and their interface will not be exposed publicly, making it very unlikely that someone would consider overriding these as a good idea.

Reply:
Works for me